commons/dev-public-catalog/configmaps/common-public-catalog-data-sql.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: common-public-catalog-data-sql
```

```suggestion
  name: common-public-catalog-data
```
it was needed in Core to distinguish between MongoDB and PGSQL